src/warnet/service_accounts.py

```diff
@@ -0,0 +1,38 @@
+import click
+
+from .constants import (
+    WARGAMES_NAMESPACE_PREFIX,
+)
+from .k8s import get_static_client
+
+
+@click.group(name="service-accounts")
+def service_accounts():
+    """Service account commands"""
+
+
+@service_accounts.command()
```

Comment:
What's the use case for having this? Is this just to avoid running a kubectl command?

Reply:
Yes. Also, if we move towards an `-o json` flag then this becomes more powerful and helpful within the code base.

Reply:
Hm, I'm not seeing how this is helpful. From what I can tell, this command would only be ran by an admin, and it's unclear what they would do with the output of this command , without needing to run more kubectl commands. If the argument is we are going to keep expanding our codebase to do k8s administrative work, I'd say this seems like we are re-writing kubectl in warnet and only for an admin user, which doesn't seem like a good idea to me.

Reply:
Sounds good. Removed it here: https://github.com/bitcoin-dev-project/warnet/pull/616/commits/d29b30e03259b2bc286f63b8b349232311fcd4c2 and https://github.com/bitcoin-dev-project/warnet/pull/616/commits/78fa9050c51395e3fa6b25ca792f3b5623e656cc